modules/spring-config/src/main/java/org/mule/runtime/config/spring/MuleArtifactContext.java

```diff
@@ -52,50 +73,44 @@
  */
 public class MuleArtifactContext extends AbstractXmlApplicationContext
 {
+
+    /**
+     * Indicates that XSD validation should be used (found no "DOCTYPE" declaration).
+     */
+    private static final int VALIDATION_XSD = 3;
```

Comment:
does the XSD frameowrk provide any constant that can be used here instead of the 3?